openassessment/templates/openassessmentblock/response/oa_response_closed.html

```diff
@@ -25,10 +24,10 @@
     <div class="wrapper--step__content">
 
         <div class="step__message message message--incomplete">
-            <h3 class="message__title">{% trans "The Due Date for This Step Has Passed" %}</h3>
+            <h5 class="message__title">{% trans "Status" %}</h5>
 
             <div class="message__content">
-              <p>{% trans "This step is now closed. You can no longer submit a response or continue with this problem, and you'll receive a grade of Incomplete. If you saved but did not submit a response, the response appears in the course records." %}</p>
+              <p>{% trans "The Due Date for This Step Has Passed. This step is now closed. You can no longer submit a response or continue with this problem, and you'll receive a grade of Incomplete. If you saved but did not submit a response, the response appears in the course records." %}</p>
```

Comment:
Make the first sentence sentence case, and delete "This step is now closed." Replace "you'll" with "you will":
"The due date for this step has passed. You can no longer submit... and you will receive..."